src/languages/lambda/code.js

```diff
@@ -0,0 +1,349 @@
+"use strict";
```

Comment:
I imagine this was probably in the source file that you copied over, but you can get rid of it because our build system adds it automatically.